pkg/volume/hostpath/host_path_test.go

```diff
@@ -154,30 +154,32 @@ func TestDeleterTempDir(t *testing.T) {
 }
 
 func TestProvisioner(t *testing.T) {
-	tempPath := fmt.Sprintf("/tmp/hostpath/%s", uuid.NewUUID())
-	defer os.RemoveAll(tempPath)
-	err := os.MkdirAll(tempPath, 0750)
-	if err != nil {
-		t.Errorf("Failed to create tempPath %s error:%v", tempPath, err)
-	}
+	tempPath := fmt.Sprintf("/tmp/hostpath.%s", uuid.NewUUID())
 	plugMgr := volume.VolumePluginMgr{}
 	plugMgr.InitPlugins(ProbeVolumePlugins(volume.VolumeConfig{ProvisioningEnabled: true}),
 		nil,
 		volumetest.NewFakeVolumeHost("/tmp/fake", nil, nil))
 	spec := &volume.Spec{PersistentVolume: &v1.PersistentVolume{Spec: v1.PersistentVolumeSpec{PersistentVolumeSource: v1.PersistentVolumeSource{HostPath: &v1.HostPathVolumeSource{Path: tempPath}}}}}
```

Comment:
Can you inline `tempPath` here since it's only used once?

Reply:
done